sources/host-ctr/go.sum

```diff
@@ -359,8 +359,7 @@ github.com/dustin/go-humanize v1.0.0/go.mod h1:HtrtbFcZ19U5GC7JDqmcUSB87Iq5E25Kn
 github.com/elazarl/goproxy v0.0.0-20180725130230-947c36da3153 h1:yUdfgN0XgIJw7foRItutHYUIhlcKzcSf5vDpdhQAKTc=
 github.com/elazarl/goproxy v0.0.0-20180725130230-947c36da3153/go.mod h1:/Zj4wYkgs4iZTTu3o/KG3Itv/qCCa8VVMlb3i9OVuzc=
 github.com/emicklei/go-restful v0.0.0-20170410110728-ff4f55a20633/go.mod h1:otzb+WCGbkyDHkqmQmT5YD2WR4BBwUdeQoFo8l/7tVs=
-github.com/emicklei/go-restful v2.16.0+incompatible h1:rgqiKNjTnFQA6kkhFe16D8epTksy9HQ1MyrbDXSdYhM=
-github.com/emicklei/go-restful v2.16.0+incompatible/go.mod h1:otzb+WCGbkyDHkqmQmT5YD2WR4BBwUdeQoFo8l/7tVs=
+github.com/emicklei/go-restful v2.9.5+incompatible/go.mod h1:otzb+WCGbkyDHkqmQmT5YD2WR4BBwUdeQoFo8l/7tVs=
```

Comment:
Hmmm, this line looks like it's adding `github.com/emicklei/go-restful v2.9.5` back into the mix.

And it does look like v2.9.5 of go restful is still being brought in somewhere in containerd:
https://github.com/containerd/containerd/blob/2806fc1057397dbaeefbea0e4e17bddfbd388f38/go.sum#L352-L353

Where and how this is being pulled in on the upstream requires more investigation upstream.

Reply:
😞 it seems to be present even in 1.7.0. https://github.com/containerd/containerd/blob/v1.7.0/go.sum

Reply:
Hmmm yeah, just noticed that as well: https://github.com/containerd/containerd/issues/8168

We'll want to follow that upstream. But for now, I'll close this and we can keep an eye on what containerd does

Reply:
@gazal-k - if you want to open a subsequent PR for just the containerd upgrades (without replacing go-restful), I'd be happy to review that